docs/src/main/asciidoc/amazon-lambda-http.adoc

```diff
@@ -222,6 +222,53 @@ HTTP response messages and the `sam.yaml` file must configure the API Gateway to
         - "*/*"
 ----
 
+== Injectable AWS Context Variables
+
+If you are using Resteasy and JAX-RS, you can inject various AWS Context variables into your JAX-RS resource classes
+using the JAX-RS `@Context` annotation.
+
+For the AWS HTTP API you can inject these AWS variables:
+[source, java]
+----
+import javax.ws.rs.core.Context;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPEvent;
+
+
+@Path("/myresource")
+public class MyResource {
+    @GET
+    public String ctx(@Context com.amazonaws.services.lambda.runtime.Context ctx) { }
+
+    @GET
+    public String event(@Context APIGatewayV2HTTPEvent event) { }
+
+    @GET
+    public String requestContext(@Context APIGatewayV2HTTPEvent.RequestContext req) { }
+
+
+}
+----
+
+For the AWS REST API we have a custom Quarkus class that represents the HTTP event.  These
```

Comment:
Maybe it would be nice to mention what custom class is that? It doesn't seem to be 100% clear from the snippet below